easybuild/easyconfigs/a/AdapterRemoval/AdapterRemoval-2.3.2-foss-2020b.eb

```
@@ -0,0 +1,39 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'ConfigureMake'
+
+name = 'AdapterRemoval'
+version = '2.3.2'
+
+github_account = 'MikkelSchubert'
+homepage = 'https://github.com/%/(github_account)s/%(namelower)s'
+description = """AdapterRemoval searches for and removes remnant adapter sequences
+ from High-Throughput Sequencing (HTS) data and (optionally) trims low quality bases
+ from the 3' end of reads following adapter removal."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```

This easyconfig of AdapterRemoval can use the `GCC` toolchain. The `foss` toolchain is only needed by software needing MPI or BLAS/LAPACK for instance.
```suggestion
toolchain = {'name': 'GCC', 'version': '10.2.0'}
```

---

@scicomp-moffitt thanks for the update! but I meant `GCC` instead of `GCCcore`. The reason is that using `GCC` leaves the door open if anybody wants to use `intel-compilers` with this software, which might be interesting for some. I made the switch to `GCC`.